cryptopp.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: cryptopp
+  version: "8.9.0"
+  epoch: 0
+  description: "Seastar is a high-performance, open-source framework for building C++ applications"
+  copyright:
+    - license: "Apache-2.0"
+
+environment:
+  contents:
+    packages:
+      - bash
+      - bazel-6
+      - binutils
+      - bison
+      - boost-dev
+      - boost-dev
+      - build-base
+      - busybox
+      - c-ares-dev
+      - ca-certificates-bundle
+      - clang-17
+      - clang-17-dev
+      - cmake
+      - coreutils # for GNU install
+      - git
+      - glibc-locale-en
+      - gnutls-dev
+      - isl-dev
+      - libtool
+      - llvm-libcxx-17
+      - llvm-libcxx-17-dev
+      - llvm-libcxxabi-17
+      - llvm-lld-17
+      - llvm17
+      - llvm17-dev
+      - openjdk-11-default-jdk
+      - openssl-dev
+      - patch
+      - py3-absl-py
+      - python3-dev
+      - samurai
+      - zlib-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: '^(\d+)\.(\d+)\.(\d+)$'
+    replace: 'CRYPTOPP_${1}_${2}_${3}'
+    to: mangled-package-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: "https://github.com/weidai11/cryptopp.git"
+      tag: "${{vars.mangled-package-version}}"
+      expected-commit: "843d74c7c97f9e19a615b8ff3c0ca06599ca501b"
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    description: "Development files for cryptopp"
+    dependencies:
+      runtime:
+        - ${{package.name}}
+    pipeline:
+      - uses: split/dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
+        - uses: test/tw/ldd-check
+          with:
+            packages: ${{package.name}}
```

Comment:
It's no longer necessary to use `with:` and `packages:` here as it will be autodetected.